python/cudf/cudf/core/index.py

```diff
@@ -1391,6 +1396,11 @@ def is_categorical(self):
     def is_interval(self):
         return False
 
+    @property  # type: ignore
+    @_cudf_nvtx_annotate
+    def hasnans(self):
+        return self._column.has_nulls(include_nan=True)
```

Comment:
This looks like it returns "has NaNs OR nulls." How do we want to handle this? If we are including nulls in the definition of `hasnans`, we need to explicitly document that.

Reply:
Updated the docstring to be explicit rather than mentioning just in an example.

Reply:
Thanks!